features/claims/litigator/interim_trial_warrant_claim_draft_submit.feature

```diff
@@ -0,0 +1,67 @@
+@javascript
```

Comment:
📝  This feature was added to cover the path where **warrant fees** are required. The existent step cover the path where they're not.